### ISSUE TITLE
[Bug Report] Scan removes manually generated thumbnail

### ISSUE THREAD

**Issue by goodguygregg** [stashapp/stash#688]
**Describe the bug**
After re-generating previews and doing a scan all my previously manually generated thumbnails are gone.

**To Reproduce**
1. Go to scenes.
2. Click on any scene, press the operations button and "generate thumbnail from current", press save.
3. Go back to scenes.
4. Select the scene you changed the thumbnail on by ticking the checkbox in the upper left corner.
5. Press "..." and in the dropdown menu press "Generate...". 
6. In the generate menu tick the following items: Previews, Sprites, Markers, Overwrite existing generated files.
7. Press generate.
8. Go back to scenes and check the thumbnail on the scene that you manually changed, it is now gone.

**Expected behavior**
The expected behavior would be to not overwrite the manually generated thumbnails. Or at least add an option for it to not overwrite in the generate menu.

**Stash Version: (from Settings -> About):**
Version: | v0.2.1-42-gc327a98
Build hash: | c327a98
Build time: | 2020-07-24 00:53:18

**Desktop (please complete the following information):**
 - OS: Kubuntu 20.04 LTS
 - Browser: Firefox 78.0.2 (64-bit)

**Comment by WithoutPants** [comment]
I'm not able to reproduce this. Tried with multiple scenes and generate does not alter the screenshot images. As far as I know, the screenshot images should only ever be modified by the "Generate thumbnail..." operations, or the scan operation. 

I was able to reproduce this behaviour if I manually deleted the jpg files in the `generated/screenshots` folder for the particular scene. If the screenshot jpg is not present in that folder, then the Scan operation does regenerate it. Did you delete any files in this folder?
